packages/web/src/styles/globalStyles.ts

```diff
@@ -179,8 +179,12 @@ export const GlobalStyle = createGlobalStyle`
 
   /* Table */
 
+  .ant-table-row .ant-table-column-sort {
+    background-color: transparent;
+  }
+
   .ant-table-thead .ant-table-cell {
-    background-color: ${Colors.slate800};
+    background-color: ${Colors.slate800} !important;
```

Comment:
Should it be `!important` for all tables?

Reply:
We can remove it for now, it was needed for sorting, as otherwise the background color would have been white